integration/service/spatialindexer/ranker/rank_by_osrm_shortest_path_impl.go

```diff
@@ -105,7 +119,7 @@ func generateTableRequest(center spatialindexer.Location, targets []*spatialinde
 
 	req.Sources = append(req.Sources, strconv.Itoa(0))
 	pointsCount4Sources := 1
-	for i := startIndex; i <= endIndex; i++ {
+	for i := 0; i <= endIndex-startIndex; i++ {
 		str := strconv.Itoa(i + pointsCount4Sources)
```

Comment:
More information, please go to: https://github.com/Telenav/osrm-backend/issues/265